packages/hooks/useAppType.ts

```diff
@@ -0,0 +1,31 @@
+import AsyncStorage from "@react-native-async-storage/async-storage";
+import { useEffect, useState } from "react";
+import { Platform } from "react-native";
+
+type AppType = "normal" | "mini" | "web3Addict";
```

Comment:
we talked about calling this Mode instead of Type
```suggestion
type AppMode = "normal" | "mini" | "web3Addict";
```